nixos/doc/manual/configuration/customizing-packages.section.md

```diff
@@ -12,6 +12,29 @@ Unfortunately, Nixpkgs currently lacks a way to query available
 configuration options.
 :::
 
+::: {.note}
+However, many packages in the `nix` store come with extensions one might add.
```

Comment:
```suggestion
Alternatively, many packages come with extensions one might add.
```

Not sure why "however", as I wouldn't read this as directly related to the warning but rather to the regular prose. Also while I think this may refer to Nixpkgs, calling it Nix store (as in App Store) would be grossly misleading.